docs/data/theme/inland-water/_data.yaml

```diff
@@ -1,5 +1,5 @@
 title: Inland water
-description: Browse inland water data products.
+description: Take a look at inland water bodies and assess their water extent, frequency, and variability.
```

Comment:
These products cover more than just water bodies (e.g. also event based and ephemeral processes like inland flooding)

```suggestion
description: Explore inland water across Australia, including changes in water extent, frequency, and variability over time.
```

Reply:
@CEKrause @robbibt 

> Analyse inland water across Australia, including changes in water extent, frequency, and variability over time.